.github/workflows/template-only-scan-orphaned-infra-test-resources.yml

```diff
@@ -0,0 +1,60 @@
+name: Template Scan Orphaned Infra Test Resources
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - fix/improve-test-cleanup-process  # Temporary: for testing this PR
+  schedule:
+    # Run every day at 08:00 UTC (4:00am ET, 1:00am PT)
+    - cron: "0 8 * * *"
+
+jobs:
+  scan:
+    name: Scan for orphaned test resources
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-region: us-east-1
+          aws-access-key-id: ${{ secrets.TESTER_AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.TESTER_AWS_SECRET_ACCESS_KEY }}
+
+      - name: Scan for orphaned resources
+        id: scan
+        run: |
+          # Run in dry-run mode to see what would be deleted
+          output=$(./template-only-bin/cleanup-test-resources --dry-run --age-hours 6 2>&1)
+          echo "$output"
+
+          # Check if any resources were found (look for "Found X resources" in output)
+          # Suppress broken pipe errors from early pipe termination
+          if (echo "$output" | grep -q "Found [1-9][0-9]* resources") 2>/dev/null; then
+            # Extract resource count and project names for notification
+            resource_info=$(echo "$output" | grep -E "(Found [0-9]+ resources|Cleaning up project:|plt-tst-act-)" | head -20) 2>/dev/null
```

Comment:
This looks to be erroring?

```
=== Cleanup complete ===
grep: write error: Broken pipe
/home/runner/work/_temp/f7e47898-e742-4bd9-ae9b-997ba22d2e5e.sh: line 9: echo: write error: Broken pipe
```

https://github.com/navapbc/template-infra/actions/runs/19839044285/job/56843173457?pr=973